akka-docs/src/test/java/jdocs/stream/RateTransformationDocTest.java

```diff
@@ -211,7 +210,7 @@ public void expandShouldTrackDrift() throws Exception {
     sub.requestNext(new Pair<>(1.0, 2));
 
     pub.sendNext(2.0);
-    Await.ready(latch, Duration.create(1, TimeUnit.SECONDS));
+    Await.ready(latch, scala.concurrent.duration.Duration.create(1, TimeUnit.SECONDS));
```

Comment:
These aren't actually shown in the sample (that's only the part between the two `// #expand-drift`) so this you can leave be as it was